### PR TITLE
[DONE]Feature multiple repo

### LINKED ISSUE #32: Répartition des package dans sara_commun et sara_robot
Plusieurs packages ne sont pas dans le bon répo..
Le but est déviter d'avoir des package demandent des driver spécifique au hardware de sara quand on est en simulation sur un ordinateur autre que celui de sara.
-  roboteq
-  wm_arm
-  wm_hand
-  wm_mecanum_cmd
- wm_odometry_feedback

Ceux-ci devraient être dans sara_robot

TODO:
- [x] Creer une branch dans sara_commun
- [x] Creer une pul request dans sara_commun
- [x] Creer une branch dans sara_robot
- [x] Creer une pul request dans sara_robot
- [x] Effectuer le transfert
- [x] Vérifier qu'il est possible de builder sans sara_robot sur un desktop

### PR DESCRIPTION
**Pull request by gortium**
### TODO:
- [x] Modify sara_install to reflect change here
### Step to merge:
1. Merge the sara_install branch of the same name.
2. Merge the sara_robot branch of the same name.
3. Merge this branch.
4. Pull change on desktop and Sara
5. Modify dependencies.rosinstall to fetch the develop branch (do the same when merging to master)
6. Build

Everything should build as usual. Package moved were obviously not needed to build or simulation. 

Now desktop should only pull sara_commun and sara_simulation if needed

Robot computer should pull sara_commun and sara_robot. 
### Packages moved to sara_robot:
- roboteq
- wm_odometry_feedback
- wm_mecanum_cmd
- wm_dynamixel_controller
- wm_arm
### Deleted package
- wm_dynamic_tf
- wm_static_tf
- wm_hand
- sara_commun (metapackage)

Fix #32

### CODE REVIEW

**Review by gregoirefra — CHANGES_REQUESTED** [review]
Est-ce que tu peux décrire les étapes à suivre une fois merge pour que nos environements soit a nouveau fonctionnel parce que j'assume que ceci va changer beaucoup de chose si quelqu'un fait juste pull les changements après. Je refuse aussi le merge juste pour l'instant pas qu'il y aille un merge accidentelle

**Comment by gortium** [comment]
Ouais oublis ca, il reste du travail a faire pour bien tout démêler !

**Comment by gortium** [comment]
Donc si tout va bien, normalement sur un desktop il n'y a rien a faire sauf pull les change de sara_commun.
Sur SARA, seulement pull les change de sara_commun et sara_robot. les deux devant être dans src.
Dans les deux cas je deleterais devel/ et build/ pour faire une catkin_make propre et assurer que ca marche.

**Comment by gortium** [comment]
@gregoirefra Commence ici et suis les instruction pour multipleRepo.

File: wm_2dnav/config/base_local_planner_params.yaml [hunk 1]
```diff
@@ -9,20 +9,20 @@ TrajectoryPlannerROS:
   acc_lim_x: 1.0
   acc_lim_y: 1.0
 
-  escape_vel: -0.2
+  escape_vel: 0.2
```
Comment (gregoirefra) [inline wm_2dnav/config/base_local_planner_params.yaml:12]: Are those change related to the new repo structure or have been introduced by mistake?

File: wm_2dnav/config/base_local_planner_params.yaml [hunk 2]
```diff
@@ -9,20 +9,20 @@ TrajectoryPlannerROS:
...
 
-  holonomic_robot: false
-  # y_vels: [-0.4, -0.1, 0.1, 0.4]
+  holonomic_robot: true
```
Comment (gregoirefra) [inline wm_2dnav/config/base_local_planner_params.yaml:14]: Are those change related to the new repo structure or have been introduced by mistake?

File: wm_2dnav/config/base_local_planner_params.yaml [hunk 3]
```diff
@@ -9,20 +9,20 @@ TrajectoryPlannerROS:
...
+  y_vels: [-0.4, -0.2, 0.2, 0.4]
```
Comment (gregoirefra) [inline wm_2dnav/config/base_local_planner_params.yaml:15]: Are those change related to the new repo structure or have been introduced by mistake?

File: wm_2dnav/config/base_local_planner_params.yaml [hunk 4]
```diff
@@ -9,20 +9,20 @@ TrajectoryPlannerROS:
...
 
  # goal tolerance parameters
   yaw_goal_tolerance: 0.10
   xy_goal_tolerance: 0.20
   latch_xy_goal_tolerance: false
 
  # forward simulation parameters
-  sim_time: 1.8
+  sim_time: 0.5
```
Comment (gregoirefra) [inline wm_2dnav/config/base_local_planner_params.yaml:23]: Are those change related to the new repo structure or have been introduced by mistake?

File: wm_2dnav/config/base_local_planner_params.yaml [hunk 5]
```diff
@@ -9,20 +9,20 @@ TrajectoryPlannerROS:
...
   sim_granularity: 0.025    # default = 0.025
-  vx_samples: 10 # default: 3
+  vx_samples: 5 # default: 3
```
Comment (gregoirefra) [inline wm_2dnav/config/base_local_planner_params.yaml:25]: Are those change related to the new repo structure or have been introduced by mistake?

File: wm_2dnav/config/base_local_planner_params.yaml [hunk 6]
```diff
@@ -33,10 +33,15 @@ TrajectoryPlannerROS:
   #The weighting for how much the controller should stay close to the path it was given. default 0.6
   pdist_scale: 0.3
   # The weighting for how much the controller should attempt to avoid obstacles. default 0.01
-  occdist_scale: 0.7
+  occdist_scale: 0.2
```
Comment (gregoirefra) [inline wm_2dnav/config/base_local_planner_params.yaml:36]: Are those change related to the new repo structure or have been introduced by mistake?

File: wm_2dnav/config/base_local_planner_params.yaml [hunk 7]
```diff
@@ -33,10 +33,15 @@ TrajectoryPlannerROS:
...
   # The weighting for how much the controller should attempt to reach its local goal, also controls speed. default 0.8
   gdist_scale: 0.5
   heading_scoring: false
   oscillation_reset_dist: 0.10
-   
-  
+
+  # Clean up the points
+  prune_plan: true
```
Comment (gregoirefra) [inline wm_2dnav/config/base_local_planner_params.yaml:43]: Are those change related to the new repo structure or have been introduced by mistake?

File: wm_2dnav/config/costmap_common_params.yaml [hunk 8]
```diff
@@ -1,11 +1,11 @@
 obstacle_range: 4.0
 raytrace_range: 5.0
 
-# footprint: [[front left corner], [front right corner], 
+# footprint: [[front left corner], [front right corner],
 #             [rear right corner], [rear left corner]]
 footprint: [[0.40, -0.33], [0.40, 0.33], [-0.40, 0.33], [-0.40, -0.33]]
 
-inflation_radius: 0.40
+inflation_radius: 0.10
```
Comment (gregoirefra) [inline wm_2dnav/config/costmap_common_params.yaml:8]: Are those change related to the new repo structure or have been introduced by mistake?

**Review by gregoirefra — CHANGES_REQUESTED** [review]
A review from someone else like @maximest-pierre and/or @JeffCousineau is also required before merge since this is a major change.

**Comment by gortium** [comment]
Jai pull master.. Mais ces change ont été introduit dans master sans passer par la branch dévelop.. Mauvaise pratique de bypass develop.. Ca donne ca.
Merge master dans develop pour le mettre a jour et ca devrais etre good.. Mais a l'avenir il faut pu bypass develop.

**Review by gregoirefra — APPROVED** [review]

**Pull request merged**